frontend/src/dispatcher/js-dispatcher.ts

```diff
@@ -20,6 +20,7 @@ export function createJsDispatcher() {
 	};
 
 	const handleJsMessage = (messageType: JsMessageType, messageData: Record<string, unknown>, wasm: WasmInstance, instance: RustEditorInstance): void => {
+		// TODO: Provide an explanatory comment here
 		const messageConstructor = messageConstructors[messageType];
```

Comment:
I think it might be worth mentioning that a message constructor does not have to be a class constructor but can also be a function in the case of `DisplayDocumentLayerTreeStructure`. On second thought it might make sense to change `messageConstructor` to `messageMaker` since the returned type is of the map is `MessageMaker`